src/psyclone/psyir/transformations/reference2arrayrange_trans.py

```diff
@@ -133,6 +133,7 @@ def validate(self, node, options=None):
             symbol.
 
         '''
+        # TODO issue #1858. Add support for structures containing arrays.
```

Comment:
Sorry to be picky but I think we should mention this limitation in the docstring of the transformation (plus the associated issue number).

Reply:
Added to the docstring.